cmd/buildtools/utils.go

```
@@ -100,6 +101,49 @@ func GetWolfiPackageVersion(wolfiAPKIndex []byte, pkgName string, pinnedVersion
 	return fmt.Sprintf("%s-r%d", pinnedVersion, revisions[0]), nil
 }
 
+func ApkoLogin() error {
+	if err := RunCommand("make", "apko"); err != nil {
+		return fmt.Errorf("make apko: %w", err)
+	}
+	if os.Getenv("REGISTRY_PASS") != "" {
+		if err := RunCommand(
+			"make",
+			"apko-login",
+			fmt.Sprintf("REGISTRY=%s", os.Getenv("REGISTRY_SERVER")),
+			fmt.Sprintf("USERNAME=%s", os.Getenv("REGISTRY_USER")),
+			fmt.Sprintf("PASSWORD=%s", os.Getenv("REGISTRY_PASS")),
+		); err != nil {
+			return fmt.Errorf("apko login: %w", err)
+		}
+	}
+	return nil
+}
+
+func ApkoBuildAndPublish(componentName string, packageVersion string) error {
+	if err := RunCommand(
+		"make",
+		"apko-build-and-publish",
+		fmt.Sprintf("IMAGE=%s/replicated/ec-%s:%s", os.Getenv("REGISTRY_SERVER"), componentName, packageVersion),
+		fmt.Sprintf("APKO_CONFIG=%s", filepath.Join("deploy", "images", componentName, "apko.tmpl.yaml")),
+		fmt.Sprintf("PACKAGE_VERSION=%s", packageVersion),
+	); err != nil {
+		return fmt.Errorf("build and publish apko for %s: %w", componentName, err)
```

i think you should just return the error rather than wrap as its repetitive